cardano-cli/test/data/golden/alonzo/genesis/verification.key

```diff
@@ -0,0 +1,5 @@
+{
+    "type": "GenesisVerificationKey_ed25519",
+    "description": "Genesis Verification Key",
```

Comment:
We should remove these keys and use them directly from `/home/jordan/Repos/Work/cardano-haskell/cardano-node/cardano-cli/test/data/golden/shelley/keys/genesis_keys/verification_key`

Likewise for the signing key. We're duplicating things here.

Reply:
I think making tests independent is a good idea. But in this case I see no problem in reusing keys.